CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -78,7 +78,7 @@ The Egress feature is graduated from Alpha to Beta and is therefore enabled by d
 - Fix CT mark matching without range in flow exporter. ([#3348](https://github.com/antrea-io/antrea/pull/3348), [@hongliangl])
 - [Windows] Enable IP forwarding of the Windows bridge local interface to fix support for Service of type LoadBalancer. ([#3137](https://github.com/antrea-io/antrea/pull/3137), [@hongliangl])
 
-[Antrea Multi-cluster]: https://github.com/antrea-io/antrea/blob/v1.6.0/docs/multicluster/getting-started.md
+[Antrea Multi-cluster]: https://github.com/antrea-io/antrea/blob/v1.6.0/docs/multicluster/user-guide.md
```

Comment:
@antoninbas The link is not available until the tag is created, it's hard to update and verify them manually. Should we consider replacing tag with release and make the job "Verify docs and spelling" verify it, assuming it's unlikely to remove a link or a section in a patch release?

Reply:
> Should we consider replacing tag with release

I assume you mean "release branch" here?
That works for me. It actually makes more sense IMO since we have a common CHANGELOG file for all patch releases and a link like this one could potentially be used for all of them.
But one issue with running the job as you suggest is that links in old CHANGELOGs can sometimes go stale. I don't know if we need to go in and change them. Ideally we would only check the "current" CHANGELOG for the release branch being tested.

Reply:
> I assume you mean "release branch" here? 

Yes.

> That works for me. It actually makes more sense IMO since we have a common CHANGELOG file for all patch releases and a link like this one could potentially be used for all of them. 

Good point.

> But one issue with running the job as you suggest is that links in old CHANGELOGs can sometimes go stale. I don't know if we need to go in and change them. Ideally we would only check the "current" CHANGELOG for the release branch being tested.

https://github.com/antrea-io/antrea/pull/3553 may resolve this.